model/docs/example_with_datasets.qmd

```diff
@@ -165,9 +165,9 @@ I0 = InfectionSeedingProcess(
 # Generation interval and Rt
 gen_int = deterministic.DeterministicPMF(gen_int, name="gen_int")
 rtproc = process.RtRandomWalkProcess(
-    Rt0_dist = dist.TruncatedNormal(loc=1.2, scale=0.2, low=0),
-    Rt_transform = transformation.ExpTransform().inv,
-    Rt_rw_dist = dist.Normal(0, 0.025),
+    Rt0_dist=dist.TruncatedNormal(loc=1.2, scale=0.2, low=0),
+    Rt_transform=transformation.ExpTransform().inv,
+    Rt_rw_dist=dist.Normal(0, 0.025),
```

Comment:
This looks fine but I want to know if this was done by `pre-commit` hooks or if you are intentionally following "no spaces padding =" as a personal norm.

Reply:
It was not done by `pre-commit` (see https://github.com/CDCgov/multisignal-epi-inference/issues/159), but it was done by my editor (VS Code).

Reply:
Per the [development standards](https://github.com/cdcent/cfa-multisignal-renewal/blob/main/ADR/model/development_standards.md#developing-practices), we follow [PEP 8](https://peps.python.org/pep-0008/) so we [should not have space padding](https://peps.python.org/pep-0008/#other-recommendations) when keyword arguments are assigned in a call.

Reply:
@dylanhmorris 

I felt pretty sure `black` was doing this by default, so wasn't expecting to see it as an addition in the PR. There is the possibility that `ruff` followed by `black` in the `pre-commit` config file undoes the change made by `black`. I need to check though and find the issue on the `black` home repo. for others encountering situations where their `black` changes have been reversed by another linter or `pre-commit` hook.

Reply:
ruff and black formatting from pre-commit do not apply to `qmd` documents. You have to open the file in vscode and format the specific code cell. I must have missed this one at some point, or someone else edited it and did not have formatting enabled in their editor.